pkg/cmd/registry/artifact/migrate/export.go

```diff
@@ -0,0 +1,103 @@
+package migrate
+
+import (
+	"context"
+	"errors"
+	"io"
+	"os"
+
+	"github.com/redhat-developer/app-services-cli/pkg/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/serviceregistry/registryinstanceerror"
+
+	"github.com/redhat-developer/app-services-cli/pkg/iostreams"
+
+	"github.com/redhat-developer/app-services-cli/pkg/logging"
+
+	"github.com/spf13/cobra"
+
+	"github.com/redhat-developer/app-services-cli/internal/config"
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/factory"
+)
+
+type ExportOptions struct {
+	file       string
+	registryID string
+
+	IO         *iostreams.IOStreams
+	Config     config.IConfig
+	Connection factory.ConnectionFunc
+	Logger     logging.Logger
+	localizer  localize.Localizer
+}
+
+func NewExportCommand(f *factory.Factory) *cobra.Command {
+	opts := &ExportOptions{
+		IO:         f.IOStreams,
+		Config:     f.Config,
+		Connection: f.Connection,
+		Logger:     f.Logger,
+		localizer:  f.Localizer,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "export",
+		Short:   f.Localizer.MustLocalize("artifact.cmd.export.description.short"),
+		Long:    f.Localizer.MustLocalize("artifact.cmd.export.description.long"),
+		Example: f.Localizer.MustLocalize("artifact.cmd.export.example"),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.registryID != "" {
+				return runExport(opts)
+			}
+
+			cfg, err := opts.Config.Load()
+			if err != nil {
+				return err
+			}
+
+			if !cfg.HasServiceRegistry() {
+				return errors.New(opts.localizer.MustLocalize("artifact.cmd.common.error.noServiceRegistrySelected"))
+			}
+
+			opts.registryID = cfg.Services.ServiceRegistry.InstanceID
+			return runExport(opts)
+		},
+	}
+	cmd.Flags().StringVar(&opts.file, "file", "", opts.localizer.MustLocalize("artifact.common.file.location"))
```

Comment:
Should it be `--output-file`?
Context: #1008

Reply:
@craicoverflow what would be your take on this?

Reply:
If the contents from the `export` command being saved in the file, it should be `--output-file`.

I notice this is not using ['artifact.common.message.file.location'](https://github.com/redhat-developer/app-services-cli/blob/1165a3fde7562851355f55a1940e1e059a4f4ea4/pkg/localize/locales/en/cmd/artifact.en.toml#L103) like other output file flags are using.

Reply:
In saying that it is obvious from the context this time (`service-account create` is less clear). But we should be consistent.